tests/acceptance/bootstrap/OcmContext.php

```diff
@@ -352,12 +352,14 @@ public function userHasDeletedFederatedConnectionWithUser(string $user, string $
 	/**
 	 * @param string $user
 	 * @param string $ocmUser
+	 * @param string|null $idp
 	 *
 	 * @return ResponseInterface
 	 * @throws GuzzleException
 	 */
-	public function deleteConnection(string $user, string $ocmUser): ResponseInterface {
+	public function deleteConnection(string $user, string $ocmUser, ?string $idp): ResponseInterface {
```

Comment:
make null as default and you can omit passing null when using it 
```suggestion
	public function deleteConnection(string $user, string $ocmUser, string $idp = null): ResponseInterface {
```